README.md

```diff
@@ -55,20 +55,58 @@ func fsIdentify(id: String, userInfo: [String: Any]) {
 ```
 
 ### Creating Events:
-- [Creating Events](https://github.com/fullstorydev/ios-shoppe-demo/blob/c534166901d71f0dace44f85aee053242dd25caf/swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift#L38)
+- [Here](https://github.com/fullstorydev/ios-shoppe-demo/blob/c534166901d71f0dace44f85aee053242dd25caf/swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift#L38) is how we create events. We started with creating an enum that incumpouses the different events in the ios app. (This could be different for you based on the flow of your app.)
+```
+enum Event: String {
+    case browsing
+    case itemSelected
+    case viewCart
+    case addToCart
+    case checkout
+    case removeFromCart
+    case crash
+}
+```
+- Then we create a convenience method that takes a case and a dictionary to title the event and provide properties.
 ```
 func fsCreateEvent(event: Event, with dict: [String: Any]) {
```

Comment:
nit: fsCreateEvent maybe better named `fsSendEvent`?

Reply:
Great idea! [CH Ticket](https://app.clubhouse.io/fullstory/story/121732/re-name-fscreateevent-to-fsevent) for later.